styles.css

```diff
@@ -583,15 +588,104 @@ ul:nth-child(4) {
   cursor: pointer;
 }
 
+.desktop-title {
+  display: none;
+}
+
+.popdiv1 {
+  position: fixed;
+  bottom: 5%;
+  left: 2%;
+  right: 2%;
+  top: 5%;
+  z-index: 999;
+  display: grid;
+  grid-template-areas:
+    "a a"
+    "b b"
+    "c c"
+    "d d"
+    "e e";
+  padding: 16px;
+  gap: 12px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  max-height: fit-content;
+}
+
+.h2-pop {
+  grid-area: a;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
```

Comment:
- [ ] Good job so far 👏 In addition to what you have done, I would suggest you add the blurred light blue background below the popup window. Apart from the fact that you have to stick to the design given, it also helps to bring focus on the popup window only 😉 

**Your design**
![image](https://user-images.githubusercontent.com/90222110/188993888-7d79e895-7fda-48ac-9937-cda5f8331b5f.png)

**Figma design**
![image](https://user-images.githubusercontent.com/90222110/188993978-aeea2d70-6b55-40b6-913e-584803e1543e.png)

Reply:
Thank you, added!